README.md

```diff
@@ -29,75 +35,20 @@ Instead, create your own new node project, and install the oss-mariner package v
 ### Step-by-step
 
 1. Create a new project folder and use `npm init` to make it a node project.
-1. Copy the contents of `runFasterCode.ts` into `index.js` in the new project.
-   1.1. <https://github.com/indeedeng/Mariner/blob/master/examples/runFasterCode.ts>
+1. Copy the contents of `runFasterCode.ts` into `index.js` and copy `config.json`, `exampleData.json`
+   in the new project. - <https://github.com/indeedeng/Mariner/blob/master/examples/runFasterCode.ts> - <https://github.com/indeedeng/Mariner/blob/master/examples/config.json> - <https://github.com/indeedeng/Mariner/blob/master/examples/exampleData.json>
 1. Comment out the existing line that imports mariner.
 1. Uncomment the line saying how mariner would normally be imported.
-1. Convert the TypeScript code to JavaScript by
-   1.1. Remove the `public` keywords from class members.
-   1.1. Remove the `implements Xxxx` from the FancyLogger class declaration.
-   1.1. Remove all the type declarations (like `: string`).
-1. Replace the path.join lines with simple hard-coded filenames: `exampleData.json` and `output.json`.
-1. Create an exampleData.json file or copy it in from Mariner.
+1. Mariner supports TypeScript, but we don't have step-by-step instructions for the TypeScript example.
+   For now, you can convert the runFasterCode.ts example file to JavaScript:
```

Comment:
I just realized that by now it's called index, so let's just say "...you can convert the example code to JavaScript"